README.md

```diff
@@ -89,4 +84,22 @@ mkvirtualenv -p /usr/local/bin/python3 notifications-template-preview
 pip install -r requirements.txt
 ```
 
-Then to run, activate the virtualenv and call `./scripts/run_app.sh 6013`
+Then create a `version.py` file under the app folder, you can copy `version.py.dist` to `version.py` to get it running.
```

Comment:
`make _generate-version-file` will run automatically... if you run one of the commands that depends on it. this is designed to be run using the makefile commands to control it through docker, it's run in docker in the real world, using a different version of python (3.6) to everything else, using particular versions of binary dependencies that aren't going to be the same as what you get on brew - i wouldn't really recommended running locally, i just threw in those lines for the curious